src/main/java/core/service/CarServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+package core.service;
+
+import core.dao.CarDao;
+import core.dao.DriverDao;
+import core.lib.Inject;
+import core.lib.Service;
+import core.model.Car;
+import core.model.Driver;
+import java.util.List;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Inject
+    private DriverDao driverDao;
```

Comment:
we don't use here driverDao, so let's remove it

Reply:
done